### ISSUE TITLE
"null is not an object" Clocklet not rendering

### ISSUE THREAD

**Issue by SterlingButters** [luncheon/clocklet#9]
Clocklet was working perfectly in CodePen, after moving into project, I keep getting the following console error:

```
[Error] TypeError: null is not an object (evaluating 'document.body.addEventListener')
	(anonymous function) (clocklet@0.2.4:1:30584)
	(anonymous function) (clocklet@0.2.4:1:159)
	Global Code (clocklet@0.2.4:1:162)
```

Any ideas?

**Comment by SterlingButters** [comment]
Ahh just needed to move script import to body - thoughts to why would be appreciated @luncheon

**Comment by luncheon** [comment]
It seems to be same as #5, fixed at v0.2.5.

**Comment by SterlingButters** [comment]
Ahh yes and I was still using v0.2.4 - Thanks again lol
